docs/src/main/sphinx/develop/client-protocol.rst

```diff
@@ -0,0 +1,257 @@
+======================
+Trino client REST API
+======================
+
+The Trino client allows users to submit Trino queries and view results.  This
+note documents the REST API used by the Trino client.
+
+HTTP methods
+============
+
+* A ``POST`` to ``/v1/statement`` runs the query string in the ``POST`` body,
+  and returns a JSON document containing the query results.  If there are more
+  results, the JSON document will contain a ``nextUri`` URL attribute.
+* A ``GET`` to the ``nextUri`` attribute returns the next batch of query results.
+* A ``DELETE`` to ``nextUri`` terminates a running query.
+
+Overview of query processing
+============================
```

Comment:
This and all of these other titles need to be changed to use dashes.. 

```
Overview of query processing
------------------------------
```

Reply:
Replaced all.